microshift_install/microshift-embed-into-rpm-ostree.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="microshift-embed-into-rpm-ostree"]
+= Embedding {product-title} into a {op-system-ostree} image
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-embed-into-rpm-ostree
+
+toc::[]
+
+You can embed {product-title} into a {op-system-ostree-first} {op-system-version} image. Use the content within this section to build a {op-system-ostree-first} {op-system-version} image containing {product-title}.
+
+[IMPORTANT]
+====
+Red Hat will not provide or support an update or upgrade path from Developer Preview and Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
+====
+
+[id="preparing-for-image-building_{context}"]
+== Preparing for image building
+
+Familiarize yourself with the documentation on https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images[Composing, installing, and managing RHEL for Edge images].
+
+[IMPORTANT]
+====
+{product-title} deployments have only been tested with {op-system} 8.7. Other versions of {op-system} are not recommended.
```

Comment:
Shouldn't the OS version also come from a variable?

Reply:
Yes. It was hard-coded there, I wasn't sure if it was because OCP 4.12 would roll up to a higher version?

Reply:
In one place we mention RHEL for Edge, which is the {op-system-ostree-first} attribute with the version attribute {op-system-version}. This note, however, is just RHEL {op-system}, not RHEL for Edge.

Reply:
The version of Image Builder OS is linked with the OS version supported by the OCP, so I think it's appropriate to use a variable here. 
Also, {op-system-version} is used subsequently in the OS version context, so these need to be in sync